mindmeld/tokenizer.py

```diff
@@ -268,8 +276,7 @@ def tokenize(self, text, keep_special_chars=True):
             else:
                 norm_token_text = self.multiple_replace(norm_token_text, self.compiled)
 
-            # fold to ascii
-            norm_token_text = self.fold_str_to_ascii(norm_token_text)
+            norm_token_text = self._normalizer.normalize(norm_token_text)
```

Comment:
To confirm, this is phase #1, where normalization happens in tokenization. But this will be changed in the future? If so, please add a TODO here about that.